tf/scripts/backend_user_data.sh

```
@@ -5,6 +5,36 @@ sudo yum install -y docker
 # Iniciar el servicio de Docker
 sudo service docker start
 
+# Instalar el agente de Datadog
+DD_API_KEY="${datadog_api_key}" DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
```

Shellcheck (0.11.0)

[warning] datadog_api_key is referenced but not assigned.

(SC2154)